src/js/post-911-gib-status/utils/helpers.js

```diff
@@ -12,6 +12,11 @@ export function formatPercent(percent) {
   return validPercent;
 }
 
+export function formatVAFilNumber(number) {
+  const lengthOfXString = number.length - 4;
```

Comment:
`va_file_number` may be `null`.  Is that accounted for at some earlier place?  Otherwise, this will throw a:
```
Uncaught TypeError: Cannot read property 'length' of null
```